easybuild/easyconfigs/g/gretl/gretl-2021c-foss-2020b.eb

```
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Loris Bennett
+# ZEDAT, Freie Universitaet Berlin
+
+easyblock = 'ConfigureMake'
+
+name = 'gretl'
+version = '2021c'
+
+homepage = 'http://gretl.sourceforge.net'
+description = "A cross-platform software package for econometric analysis"
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'usempi': True}
+
+# http://prdownloads.sourceforge.net/gretl/gretl-2020a.tar.xz
+source_urls = ['http://prdownloads.sourceforge.net/%(name)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['be468b8233b2702a15df24d1efffdf44ce082c99617cbfa01d8e2fafda370f8a']
+
+builddependencies = [
+    ('gnuplot', '5.4.1'),
+    ('libxml2', '2.9.10'),
+    ('GMP', '6.2.0'),
+    ('GtkSourceView', '3.24.11'),
+    ('ImageMagick', '7.0.10-35'),
+    ('MPFR', '4.1.0'),
+]
+
+
+preconfigopts = 'export LDFLAGS="-fopenmp $LDFLAGS" && '
+
+configopts = '--disable-json '
+
+sanity_check_paths = {
+    'files': ['bin/gretl', 'bin/gretlcli'],
+    'dirs': ['bin', 'share']
+}
+
```

It would also be good to add one or more commands as `sanity_check_commands` here, e.g. `gretl --help` or `gretl --version`. I've just done a build with `cURL` added to the build deps, and even though the installation succeeded, `gretl --help` still failed: it fails to find the `ncurses` and `readline` libraries, which will have to be added as dependencies. A sanity check command would help spotting these issues.

```suggestion

sanity_check_commands = [
    'gretl --help',
    'gretlmpi --help',
]

```